torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12286,6 +12296,20 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(unittest.expectedFailure, 'TestMeta', 'test_dispatch_meta_outplace', device_type='cuda'),
                DecorateInfo(unittest.expectedFailure, 'TestMeta', 'test_dispatch_symbolic_meta', device_type='cuda'),
            )),
+    OpInfo('view_copy',
```

Comment:
If there was no opinfo coverage before, then there is likely some manual testing done somewhere. 

Have to checked if any of the existing manual tests are redundant and can be removed?

Reply:
I didn't know about the manual tests. can you show me where to look?

Reply:
`git grep view_copy -- "test/*"` turns up quite a few hits. I suspect some of the stuff from `test/test_view_ops.py` possibly some of the stuff in `test/test_autograd.py` might be redundant with the coverage from the opinfo tests. 

Feel free to ask about specific items if you are unsure.  If there is nothing obvious it is always safe to leave the existing tests. Not something you need to spend a lot of time on.

Reply:
Ok I removed a few, but kept the test `test/test_jit_cuda_fuser.py:test_view_copy_graph_guard` because I was unsure of what it was doing.